dev/com.ibm.ws.security.jaas.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -35,6 +35,10 @@ jaasLoginModule.desc=A login module in the JAAS configuration.
 className=Class name
 className.desc=Fully-qualified package name of the JAAS login module class.
 
+classProviderRef=Class Provider
+classProviderRef.desc=A reference to the ID of the resource adapter or application that provides the JAAS login module class. Only specify this configuration attribute if you do not specify a shared library reference.
```

Comment:
**change**
Only specify this configuration attribute if you do not specify a shared library reference.

**to**
Specify this configuration attribute only if you do not specify a shared library reference.

Reply:
Thanks - I added commit 
https://github.com/OpenLiberty/open-liberty/pull/10885/commits/b72e9c77050c1b12b9d858e0901e3b0f0462ba58
with the requested change.  Please approve the review if I've copied it over correctly.